components/ContactUsForms/InterestForm/InterestForm.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <el-form
+    ref="submitForm"
+    label-position="top"
+    :model="form"
+    :rules="formRules"
+    :hide-required-asterisk="true"
+  >
+    <el-form-item
+      class="service-categories vertical-content mt-32"
+      prop="serviceCategories"
+      label="What service(s) are you interested in? *"
+    >
+      <div class="body4 mb-4"><i>Select all that apply.</i></div>
+      <client-only>
+        <sparc-checkbox
+          v-for="service in services"
+          v-model="form.serviceCategories"
+          :key="service"
+          :label="service"
+          :display="service"
+        />
+      </client-only> 
+    </el-form-item>
+
+    <el-form-item prop="additionalInfo" label="Additional Information">
+      <el-input
+        v-model="form.additionalInfo"
+        type="textarea"
+        :rows="3"
+        placeholder="Please provide any additional information regarding your service request"
+      />
+    </el-form-item>
+
+    <hr/>
+
+    <user-contact-form-item showFollowUpOption v-model="form.user" />
+
+    <hr/>
+
+    <div class="heading2">
+      Please check the box to proceed
+    </div>
+
+    <el-form-item prop="recaptcha">
+      <recaptcha-checkbox v-model="form.recaptcha" class="recaptcha my-16 pl-16"/>
+    </el-form-item>
+
+    <hr/>
+
+    <el-form-item>
+      <el-button class="primary" :disabled="isSubmitting" @click="onSubmit">
+        Submit
+      </el-button>
+      <p v-if="hasError" class="error">
+        An error has occurred, please try again.
+      </p>
+    </el-form-item>
+  </el-form>
+</template>
+
+<script>
+import NewsletterMixin from '../NewsletterMixin'
+import RecaptchaMixin from '@/mixins/recaptcha/index'
+import UserContactFormItem from '../UserContactFormItem.vue'
+import { isEmpty } from 'ramda'
+// import { mapState, mapGetters } from 'vuex'
```

Comment:
Please update all the references to Vuex to use Pinia (That is the new recommended store that we are using). You can find many examples of usages throughout the code. The store index file is found at store/index